contracts/bprotocol/avatar/Avatar.sol

```diff
@@ -64,4 +65,26 @@ contract Avatar is AbsComptroller, AbsCToken {
         uint256 result = super.mint(cToken, mintAmount);
         return result;
     }
+
+    // EMERGENCY FUNCTIONS
+    function resetApprove(IBToken bToken) external onlyAvatarOwner {
```

Comment:
if this is under emergency, maybe just let it reset any token.
it will take an IERC20 as input, and will reset it.